src/sentry/static/sentry/app/views/performance/landing/content.tsx

```diff
@@ -0,0 +1,219 @@
+import React from 'react';
+import {browserHistory, withRouter, WithRouterProps} from 'react-router';
+import styled from '@emotion/styled';
+import {Location} from 'history';
+
+import Feature from 'app/components/acl/feature';
+import DropdownControl, {DropdownItem} from 'app/components/dropdownControl';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization, Project} from 'app/types';
+import EventView from 'app/utils/discover/eventView';
+import {generateAggregateFields} from 'app/utils/discover/fields';
+import {stringifyQueryObject, tokenizeSearch} from 'app/utils/tokenizeSearch';
+import SearchBar from 'app/views/events/searchBar';
+
+import Charts from '../charts/index';
+import {generateFrontendPerformanceEventView} from '../data';
+import Table from '../table';
+import {getTransactionSearchQuery} from '../utils';
+
+import {FRONTEND_COLUMN_TITLES} from './data';
+import FrontendDisplay from './frontendDisplay';
+import {
+  getAdditionalTableQuery,
+  getCurrentLandingDisplay,
+  LANDING_DISPLAYS,
+  LandingDisplayField,
+} from './utils';
+import VitalsCards from './vitalsCards';
+
+type Props = {
+  organization: Organization;
+  eventView: EventView;
+  location: Location;
+  projects: Project[];
+  setError: (msg: string | undefined) => void;
+  handleSearch: (searchQuery: string) => void;
+} & WithRouterProps;
+
+type State = {};
+class LandingContent extends React.Component<Props, State> {
+  getSummaryConditions(query: string) {
+    const parsed = tokenizeSearch(query);
+    parsed.query = [];
+
+    return stringifyQueryObject(parsed);
+  }
+
+  handleLandingDisplayChange = (field: string) => {
+    const {location} = this.props;
+
+    browserHistory.push({
+      pathname: location.pathname,
+      query: {
+        ...location.query,
+        landingDisplay: field,
+      },
+    });
+  };
+
+  handleTableQueryUpdate = (additionalTableQuery: string) => {
+    const {location} = this.props;
+
+    browserHistory.push({
+      pathname: location.pathname,
+      query: {
+        ...location.query,
+        tableFilterQuery: additionalTableQuery,
+      },
+    });
+  };
+
+  render() {
+    const {
+      organization,
+      location,
+      router,
+      projects,
+      eventView,
+      setError,
+      handleSearch,
+    } = this.props;
+
+    const currentLandingDisplay = getCurrentLandingDisplay(location);
+    const filterString = getTransactionSearchQuery(location, eventView.query);
+    const summaryConditions = this.getSummaryConditions(filterString);
+
+    return (
+      <div>
+        <Feature organization={organization} features={['performance-landing-v2']}>
+          {({hasFeature}) => {
+            if (hasFeature) {
+              const additionalSummaryConditions = this.getSummaryConditions(
```

Comment:
I mean... it should still be the summary conditions since it's getting added to `query` which is then used in the table summary conditions, but agreed that it's conflated.